### PR TITLE
Add `github_actions` label file match

### PR DESCRIPTION
**Pull request by anGie44**
### Community Note

* Please vote on this pull request by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original pull request comment to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for pull request followers and do not help prioritize the request

Output from acceptance testing:

```
N/A
```

### CODE REVIEW

File: .github/labeler-pr-triage.yml [hunk 1]
```diff
@@ -17,6 +17,8 @@ generators:
   - 'internal/**/*_gen_test.go'
   - 'internal/**/generate.go'
   - 'internal/generate/**/*'
+github_actions:
+  - '.github/workflows/*.yml'
```
Comment (anGie44) [inline .github/labeler-pr-triage.yml:21]: maybe this should include the yml files in just `.github` dir?

Comment (justinretzolk) [reply .github/labeler-pr-triage.yml:21]: I think it's worth adding the yml files in the `.github` dir too, for sure.

**Review by justinretzolk — APPROVED** [review]
LGTM!

File: .github/labeler-pr-triage.yml (hunk 1 above)
Comment (ewbankkit) [reply .github/labeler-pr-triage.yml:22]: I took the liberty of add `.github/*.yml` to the list of _github_actions_ patterns.

**Review by ewbankkit — APPROVED** [review]
LGTM :rocket:.

File: .github/labeler-pr-triage.yml (hunk 1 above)
Comment (anGie44) [reply .github/labeler-pr-triage.yml:22]: thank you!

**Pull request merged**